src/engraving/libmscore/system.cpp

```diff
@@ -1824,4 +1824,24 @@ int System::lastVisibleSysStaffOfPart(const Part* part) const
     }
     return -1;   // No visible staves on this part.
 }
+
+//---------------------------------------------------------
+//      minSysTicks
+//      returns the shortest note/rest in the system
+//---------------------------------------------------------
+
+Fraction System::minSysTicks() const
+{
+    Fraction minTicks = Fraction(10, 1); // Initializing the variable at a random high value.
```

Comment:
still no explanation here

other developers need to understand what you think (10, 1) is sufficient number here

Reply:
Wait, but the variable changes its value inside this method. And it needs to start at a high value every time the method is called. So I cannot declare it static constexpr. Or am I missing something?

Reply:
The longest note/rest MuseScore can possibly create is a quadruple dotted longa, that is one quarter short of 8/1.
So `Fraction(8, 1)` is more than enough already, no need to go to `Fraction (INT_MAX, 1)`.
Well, maybe, one day, we add suppoer for a maxima (8/1), which quadrouple dotted would be 15.5/1 , so `Fraction(16/1)` should be on the safest possible side ;-)

Reply:
I have the feeling that we may be overthinking this. The fraction really just needs to be longer than any possible note/rest, but "how much longer" does not make any difference, neither conceptually nor computationally. Choosing Fraction (10, 1) was a bad idea because it looks like a "reasonable" value, so a reader would assume that it is supposed to "mean something". 
I should have started with Fraction(1000, 1) or something like that, so it's immediately obvious that it's a random value. This is what I'm doing now. And I'm adding a more verbose comment. We should be fine :)

Reply:
Well, 1000 is a much a 'magic number' as 10. Or 16. The latter at least makes sense.